### ISSUE TITLE
Link posts shouldn't have read time estimations?

### ISSUE THREAD

**Issue by berekuk** [LessWrong2/Lesswrong2#862]
...Or should they?

Some link posts include the copy of the linked post and some include just the short description, and there's no easy way to detect which is which. But for most link posts it's just misleading.

**Comment by Discordius** [comment]
Yeah, it's a bit unclear. Some linkposts are neither of those, and are
indeed a bunch of commentary on the post. It might make sense to deactivate
the reading times, but I am not yet confident.
